src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -12,127 +12,196 @@ <h2>
-                                *ngIf="exercise.isAtLeastInstructor"
-                                jhiDeleteButton
-                                [entityTitle]="participation.student.name"
-                                deleteQuestion="artemisApp.participation.delete.question"
-                                (delete)="deleteParticipation(participation.id, $event)"
-                                [additionalChecks]="
-                                    exercise.type === PROGRAMMING
-                                        ? { deleteBuildPlan: 'artemisApp.participation.deleteBuildPlan', deleteRepository: 'artemisApp.participation.deleteRepository' }
-                                        : null
-                                "
-                            >
-                                <fa-icon [icon]="'times'"></fa-icon>
-                            </button>
-                            <button
-                                type="submit"
-                                *ngIf="exercise?.type === PROGRAMMING && participation.buildPlanId != null && exercise.isAtLeastInstructor"
-                                [routerLink]="['/', { outlets: { popup: ['participation', participation.id, 'cleanupBuildPlan'] } }]"
-                                class="btn btn-danger btn-sm mr-1"
-                            >
-                                <fa-icon [icon]="'eraser'"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="entity.action.cleanupBuildPlan">Cleanup build plan</span>
-                            </button>
-                            <button
-                                *ngIf="this.presentationScoreEnabled && participation.presentationScore !== 1"
-                                (click)="addPresentation(participation)"
-                                class="btn btn-info btn-sm mr-1"
-                            >
-                                <fa-icon [icon]="'file-powerpoint'"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="artemisApp.participation.addPresentation.label">Add presentation</span>
-                            </button>
-                            <button
-                                *ngIf="this.presentationScoreEnabled && participation.presentationScore == 1"
-                                (click)="removePresentation(participation)"
-                                class="btn btn-danger btn-sm mr-1"
-                            >
-                                <fa-icon [icon]="'file-powerpoint'"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="artemisApp.participation.removePresentation.label">Remove presentation</span>
-                            </button>
+                    </ng-template>
+                </ngx-datatable-column>
+
+                <ngx-datatable-column prop="repositoryUrl" *ngIf="exercise?.type == PROGRAMMING" [minWidth]="120">
```

Comment:
Please use `===` instead. Double equals can be quit dangerous

Reply:
Fixed it.